lib/rust/ensogl/app/theme/hardcoded/src/lib.rs

```diff
@@ -650,6 +650,9 @@ define_themes! { [light:0, dark:1]
         slider {
             background = graph_editor::node::background , graph_editor::node::background;
             overshoot_limit = 60.0, 60.0;
+            text {
+                font = "default", "default";
+            }
```

Comment:
Let's remove this. It just doesnt make sense to have a specific per-slider font setting in a theme.